libretro-rs/src/lib.rs

```diff
@@ -333,27 +348,31 @@ pub enum RetroJoypadButton {
   R2 = 13,
   L3 = 14,
   R3 = 15,
+  #[cfg(experimental)]
+  MASK = 256,
```

Comment:
Casing here doesn't match line 375 (`MASK` here, `Mask` there). You should be able to use the example project to enable `experimental` in its `Cargo.toml` and see if it errors.

Reply:
Good eye